### PR TITLE
add staging project provider-aws

### PR DESCRIPTION
**Pull request by andrewsykim**
We need a GCR image for https://github.com/kubernetes/cloud-provider-aws 

cc @nckturner @justinsb 

Signed-off-by: Andrew Sy Kim <kim.andrewsy@gmail.com>

### CODE REVIEW

**Comment by spiffxp** [comment]
LGTM but we're blocked until we can get billing project quota raised ref: ttps://github.com/kubernetes/k8s.io/issues/852

File: infra/gcp/ensure-staging-storage.sh [hunk 1]
```diff
@@ -44,6 +44,7 @@ STAGING_PROJECTS=(
     autoscaling
     build-image
     cip-test
+    cloud-provider-aws
```
Comment (dims) [inline infra/gcp/ensure-staging-storage.sh:47]: can you please rename this to `provider-aws`? it's too long otherwise

**Comment by dims** [comment]
@andrewsykim please see note inside please follow the pattern of `provider-xxxx` else the project name is too long

/hold

**Comment by andrewsykim** [comment]
@dims updated, thanks

**Comment by spiffxp** [comment]
Ran `./infra/gcp/ensure-staging.sh provider-aws`

**Pull request merged**